scripts/main.js

```diff
@@ -71,4 +71,134 @@ const departments = {
+    case "marketing":
+        console.log(`Je koos: ${departmentChoice}. ${departments.marketing.description}`);
+        document.getElementById('department-description').textContent = departments.marketing.description;
+        break;
+    case "sales":
+        console.log(`Je koos: ${departmentChoice}. ${departments.sales.description}`);
+        document.getElementById('department-description').textContent = departments.sales.description;
+        break;
+    case "customer-service":
+        console.log(`Je koos: ${departmentChoice}. ${departments["customer-service"].description}`);
+        document.getElementById('department-description').textContent = departments["customer-service"].description;
+        break;
+    default:
+        console.error(`Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.`);
+        document.getElementById('error-message').textContent = "Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.";
+}
+
+//------------------------ Opdracht 3 ------------------------------------------------
+
+// // Opdracht 3a
+//
+// const jobChoice = prompt("Je koos marketing. Over welke functie wil je meer weten? Voer een getal tussen 0 en 3 in. "
+//     + " - 0:" + departments.marketing.jobs[0].title
+//     + " - 1:" + departments.marketing.jobs[1].title
+//     + " - 2:" + departments.marketing.jobs[2].title +
+//     + " - 3:" + departments.marketing.jobs[3].title);
+//
+// console.log(jobChoice);
+//
+// // Opdracht 3b en 3c
+//
+// switch (jobChoice) {
+//     case "0":
+//         console.log("Je koos", departments.marketing.jobs[0].title, ". Een uitdagende rol!", departments.marketing.jobs[0].description);
+//         break;
+//     case "1":
+//         console.log("Je koos", departments.marketing.jobs[1].title, ". Een uitdagende rol!", departments.marketing.jobs[1].description);
+//         break;
+//     case "2":
+//         console.log("Je koos", departments.marketing.jobs[2].title, ". Een uitdagende rol!", departments.marketing.jobs[2].description);
+//         break;
+//     case "3":
+//         console.log("Je koos", departments.marketing.jobs[3].title, ". Een uitdagende rol!", departments.marketing.jobs[3].description);
+//         break;
+//     default:
+//         console.error("Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.")
+// }
+
+//------------------------------- Opdracht 4 -------------------------------------------
+
+// Opdracht 4a
+
+console.log(`${departmentChoice} is een leuke afdeling om te werken. Er werken op dit moment ${departments[departmentChoice].numberOfEmployees} medewerkers.`);
+
+// Opdracht 4b
+
+const jobChoice = prompt(`Je koos ${departmentChoice}. Over welke functie wil je meer weten? 
+Voer een getal tussen 0 en 3 in.
+* 0: ${departments[departmentChoice].jobs[0].title}
+* 1: ${departments[departmentChoice].jobs[1].title}
+* 2: ${departments[departmentChoice].jobs[2].title}
+* 3: ${departments[departmentChoice].jobs[3].title}`);
+
```

Comment:
marketing.jobs[4] kan niet opgeroepen worden. Deze bestaat wel

Dit kan in de uitwerkingen van NOVI ook niet, het is dus wel goed.